assignments/closure.js

```diff
@@ -1,6 +1,10 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+const noWorries = 'Hakuna Matata';
```

Comment:
Definitely review the closures material in the TK (watch Josh Knell's videos). Here's my solution to this closure problem:
const youCantTouchThis = function (x) {
    return function (y) {
      return x + y;
    }
}

const testName = youCantTouchThis("Joshua")(" Wood");
console.log(testName);